app/js/uiConfig.js

```diff
@@ -8,6 +8,10 @@
 const defaultConfig = {
 	homePlugin: 'Overview',
 	siadAddress: 'http://localhost:9980',
+	width: 800,
+	height: 600,
```

Comment:
does this mean the UI will default to this size for all users? I'm so used to it being fullscreen haha